vllm/worker/hpu_worker.py

```diff
@@ -203,6 +203,9 @@ def determine_num_available_blocks(self) -> Tuple[int, int]:
         num_hpu_blocks = max(num_hpu_blocks, 0)
         num_cpu_blocks = max(num_cpu_blocks, 0)
 
+        self.model_runner.bucketing_global_state.decode_block_bucket_cfg[
```

Comment:
It seems that decode_block_bucket_cfg is a tuple, you could convert to list and then to tuple back